paddle/fluid/pybind/imperative.cc

```diff
@@ -1910,54 +1910,51 @@ void BindImperative(py::module *m_ptr) {
+                 platform::errors::InvalidArgument(
+                     "Tensor %s has not been initialized!", self->Name()));
              t->clear();
            })
       .def("_offset",
            [](const std::shared_ptr<imperative::VarBase> &self) {
              auto *t = self->MutableVar()->GetMutable<framework::LoDTensor>();
-             PADDLE_ENFORCE_EQ(t->IsInitialized(), true,
-                               platform::errors::InvalidArgument(
-                                   "tensor has not been initialized"));
+             PADDLE_ENFORCE_EQ(
+                 t->IsInitialized(), true,
+                 platform::errors::InvalidArgument(
+                     "Tensor %s has not been initialized!", self->Name()));
              return t->offset();
            })
-      .def("_share_buffer_with",
+      .def("_share_buffer_to",
            [](const std::shared_ptr<imperative::VarBase> &self,
-              std::shared_ptr<imperative::VarBase> &target_t) {
-             auto *t = self->MutableVar()->GetMutable<framework::LoDTensor>();
-             auto *t_t =
-                 target_t->MutableVar()->GetMutable<framework::LoDTensor>();
-             PADDLE_ENFORCE_EQ(t->IsInitialized(), true,
-                               platform::errors::InvalidArgument(
-                                   "tensor has not been initialized"));
-             PADDLE_ENFORCE_EQ(t_t->IsInitialized(), true,
-                               platform::errors::InvalidArgument(
-                                   "tensor has not been initialized"));
-             t->ShareBufferWith(*t_t);
+              std::shared_ptr<imperative::VarBase> &dst) {
+             auto *src = self->MutableVar()->GetMutable<framework::LoDTensor>();
+             auto *dst_ = dst->MutableVar()->GetMutable<framework::LoDTensor>();
+             PADDLE_ENFORCE_EQ(
+                 src->IsInitialized(), true,
+                 platform::errors::InvalidArgument(
+                     "Tensor %s has not been initialized!", self->Name()));
+             dst_->ShareBufferWith(*src);
            })
-      .def("_is_shared_buffer_with",
+      .def("_is_shared_buffer_to",
            [](const std::shared_ptr<imperative::VarBase> &self,
-              std::shared_ptr<imperative::VarBase> &target_t) {
-             auto *t = self->MutableVar()->GetMutable<framework::LoDTensor>();
-             auto *t_t =
-                 target_t->MutableVar()->GetMutable<framework::LoDTensor>();
-             PADDLE_ENFORCE_EQ(t->IsInitialized(), true,
-                               platform::errors::InvalidArgument(
-                                   "tensor has not been initialized"));
-             PADDLE_ENFORCE_EQ(t_t->IsInitialized(), true,
-                               platform::errors::InvalidArgument(
-                                   "tensor has not been initialized"));
-             return t->IsSharedBufferWith(*t_t);
+              std::shared_ptr<imperative::VarBase> &dst) {
```

Comment:
This Api should still be `_is_shared_buffer_with` .

Reply:
Done.